### DISCUSSION TITLE
Transfer of repositories to GrandOrgue organization

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#1809] [Developers]
I've just begun the process of migrating LoopAuditioneer to GitHub. In a recent mail conversation with Oleg the topic of a possible transfer of some of the GrandOrgue related repositories that I maintain to the GrandOrgue organization came up. So I wonder, would it be ok to transfer

https://github.com/larspalo/loopauditioneer
https://github.com/larspalo/libsndfile
https://github.com/larspalo/GOODF

to GrandOrgue organization as they currently are? I realize both LoopAuditioneer and GOODF needs much more work, refinement and attention, but I think that they are very much usable and can fill a vital role for sample set producers. Any thoughts or concerns?

**Comment by rousseldenis** [comment]
@larspalo If you think moving them here should bring some visibility, why not. At least a link to them in GrandOrgue wiki should be sufficent (with maybe a dedicated page to sample set creation ?).

**Reply by larspalo** [reply]
@rousseldenis Visibility is one thing, but I mostly want to only manage repositories on one platform and GitHub does have better automated build/release options. Besides that, both LoopAuditioneer and GOODF is very much written with GrandOrgue in mind so it makes sense to keep them all under the same umbrella so to say. You're ok with the transfer then?

**Reply by rousseldenis** [reply]
Yes of course.

**Comment by oleg68** [comment]
@larspalo The only thing to discuss is the names of the repositories.

Could you place them in CamelCase? For example, LoopAuditioner, LibSndFile, GoOdf?

**Reply by larspalo** [reply]
@oleg68 LoopAuditioneer and GoOdf will certainly be good and I can change that in the transfer. I'm a bit more reluctant with libsndfile as it's a well known and used library present in many (most) Linux distributions. Changing that name feels a bit sacrilegious to be honest. Do you really think it's necessary?

**Reply by oleg68** [reply]
If libsndfile is a fork of an existing github repo then you may leave it's name unchanged.

But I hope that this repository will be temporary: untill your PR is merged upstream.
